src/pages/call/call.html

```diff
@@ -1,7 +1,53 @@
 <ion-header>
-	<header [userData]="userData"></header>
+    <header [userData]="userData"></header>
 </ion-header>
 
 <ion-content padding>
-	Call content
+
+    <ion-grid text-center>
+        <ion-row>
+            <ion-col>
+                <img src="assets/img/addcontact_btn.svg">
+            </ion-col>
+            <ion-col>
+                <h1>&nbsp;{{phoneNumber}}&nbsp;</h1>
+            </ion-col>
+            <ion-col>
+                <img src="assets/img/delete_btn.svg"
+                     (press)="removeAll()" (click)="removeLast()">
+            </ion-col>
+        </ion-row>
```

Comment:
Great, thanks Murhaf!

I've added your changes (in 47e9522) with some minor differences:
* Instead of "ios=" and "md=" I'm using "name=" because it gets the right icons already for our platforms.
* The style "phone-number" is used for the header too, so I'm using "phone-number-call" instead.

Also, I'm using your tip of "[hidden]="!phoneNumber.lengtht"" to simplify my (simple anyway) typescript code.